sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/AsyncDocumentClient.java

```diff
@@ -1479,4 +1482,11 @@ public AzureKeyCredential getCredential() {
     void close();
 
     ItemDeserializer getItemDeserializer();
+
+    /**
+     * Enable throughput control.
+     *
+     * @param groups the throughput control group set.
+     */
+    void enableThroughputControl(Set<ThroughputControlGroup> groups);
```

Comment:
can we use Iterable? why Set not Iterable?

Reply:
since we do not allow duplicate group definition, so thought set is more appropriate here